.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plugins"]
+	path = plugins
+	url = https://github.com/getpelican/pelican-plugins.git
```

Comment:
Is it a cleaner way to install the plugin in the system?  But it's OK for just do it this way for this website :)